pkg/intercp/envoyadmin/forwarding_kds_client.go

```diff
@@ -182,3 +183,11 @@ func (f *forwardingKdsEnvoyAdminClient) clientForInstanceID(ctx context.Context,
 	}
 	return f.newClientFn(instance.InterCpURL())
 }
+
+type StreamNotConnectedError struct {
+	rpcName string
+}
+
+func (e StreamNotConnectedError) Error() string {
+	return fmt.Sprintf("stream to execute %s operations is not yet connected", e.rpcName)
+}
```

Comment:
You need to define is for this your fix to work as otherwise it's going to do `==` comp and not match

Reply:
Done